src/main/scala/org/broadinstitute/dsde/workbench/avram/Avram.scala

```diff
@@ -32,14 +35,17 @@ import org.broadinstitute.dsde.workbench.avram.util.SlickDatabaseFactory
   * back to us and might be worth exploring.
   */
 object Avram {
-  private val databaseFactory = new SlickDatabaseFactory(AvramConfig.dbcpDataSourceConfig)
+  private val configFactory = ConfigFactory.parseResources("app.conf").withFallback(ConfigFactory.load())
+  private val dbcpDataSourceConfig = configFactory.as[DbcpDataSourceConfig]("dbcpDataSource")
+
+  val database =  DbReference(dbcpDataSourceConfig)
```

Comment:
This can be:
```
  val database = DbReference(AvramConfig.dbcpDataSourceConfig)
```
Then you don't need the two lines above to load the configs.

Also a nit: extra space after `=`.